Assets/Scenes/Main.unity

```diff
@@ -9118,7 +9113,7 @@ PrefabInstance:
       addedObject: {fileID: 645481521}
     - targetCorrespondingSourceObject: {fileID: 2773908478669454240, guid: 9963fec33e379324db2affcb3f83332f,
         type: 3}
-      insertIndex: 2
+      insertIndex: 1
```

Comment:
is change op de main scene nodig?

Reply:
Ik moet eerlijk bekennen dat ik niet 100% zeker weet wat dit is, maar aangezien het niet een van die positie wijzigingsdingen is maar een echte change lijkt te zijn neem ik aan dat die nodig is :)